src/mediarithmics/api/plugin/audiencefeedconnector/AudienceFeedConnectorPluginResponseInterface.ts

```diff
@@ -18,17 +20,21 @@ export interface ExternalSegmentConnectionPluginResponse {
 
 export interface UserSegmentUpdatePluginResponse {
   status: AudienceFeedConnectorStatus;
-  data?: [{
-    destination_token?: string;
-    grouping_key?: string
-    content?: string;
-    binary_content?: BinaryType;
-  }];
-  stats?: [{
-    identifier?: string;
-    sync_result?: string;
-    tags?: any
-  }];
+  data?: UserSegmentUpdatePluginResponseData[];
+  stats?: UserSegmentUpdatePluginResponseStats[];
   message?: string;
   nextMsgDelayInMs?: number;
+}
+
+export interface UserSegmentUpdatePluginResponseData {
+  destination_token?: string;
```

Comment:
This is mandatory and grouping_key too

Reply:
Not blocking but referenced here : https://github.com/MEDIARITHMICS/plugins-nodejs-sdk/issues/97